examples/README.md

```diff
@@ -0,0 +1,8 @@
+EXAMPLES
+========
+
+These files, which previously lived under the direcotry `cmd`, are examples of
```

Comment:
directory is misspelled, but it's not necessary to mention they used to be in `cmd`.